From 44fcb2ec647ba4e56ab4d3ebea42f08aa71677f5 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca.weiss@fairphone.com>
Date: Mon, 11 Mar 2024 11:02:29 +0100
Subject: [PATCH] rmtfs.service.in: Remove dependency on qrtr-ns.service

The QRTR nameserver has been built into the kernel for years now, drop
the dependency since qrtr-ns.service won't do anything anyways.
---
 rmtfs.service.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/rmtfs.service.in b/rmtfs.service.in
index 8384043..abd12df 100644
--- a/rmtfs.service.in
+++ b/rmtfs.service.in
@@ -1,7 +1,5 @@
 [Unit]
 Description=Qualcomm remotefs service
-Requires=qrtr-ns.service
-After=qrtr-ns.service
 
 [Service]
 ExecStart=RMTFS_PATH/rmtfs -r -P -s
